packages/xo-web/src/xo-app/pool/tab-patches.js

```diff
@@ -269,16 +265,14 @@ export default class TabPatches extends Component {
                           })
                         : isSingleHost
                           ? _('multiHostPoolUpdate')
```

Comment:
I read this as "if is single host do multi host pool update ? " is it ok ?

Reply:
Yes.
The message for `multiHostPoolUpdate` is: `Rolling pool update can only work when there's multiple hosts in a pool with a shared storage`.
The key doesn't really match the message, but it won't be changed in this PR